modules/python/python-dest.c

```diff
@@ -405,8 +405,6 @@ _py_init_bindings(PythonDestDriver *self)
   g_ptr_array_add(self->py._refs_to_clean, self->py.open);
   g_ptr_array_add(self->py._refs_to_clean, self->py.flush);
   g_ptr_array_add(self->py._refs_to_clean, self->py.send);
-  g_ptr_array_add(self->py._refs_to_clean, self->py.log_template_options);
-  g_ptr_array_add(self->py._refs_to_clean, self->py.seqnum);
```

Comment:
In case we store these object in `PythonDestDriver`, we should not decrement our reference counter after setatttr, because the reference added by `PyObject_SetAttrString()` can easily be decremented on the Python side by assigning a new value to that attribute. `pyseqnum` and `pylog_template_options` would point to invalid memory in that case.

I checked and we don't really use `self->py.log_template_options` and `self->py.seqnum` after the setattr call, so I think we could just remove these refs from the C struct.

Reply:
Yes-yes, good point!

Let's just make them local variables then!

Reply:
Fixed!